docs/usage-notes.md

```diff
@@ -22,6 +23,29 @@ Once the app is launched (either web or desktop version), it will prompt you to
 
 ![session_select](https://imgur.com/nBOy7zH.png)
 
+## Format of `profiles.json` 
+
+You can choose to load your custom sessions by clicking on the **file icon** beside the session dropdown. 
```

Comment:
```suggestion
If your sessions are not present in the default dropdown list on CATcher's startup page, you can load custom sessions by clicking on the **file icon** beside the session dropdown. 
```

Would be better to briefly explain why someone might want to use custom sessions.